CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Change Log
 
+**_WARNING!_ GET /api/logout endpoint is deprecated. It will work for now, 
+  but you should use POST instead. See #100**
+
+## Changed
+
+#### Fixed
+- Issue #100: /api/logout endpoint should be POST
```

Comment:
Since `GET` was the intended behavior (but just bad practice), this should be listed under the "Changed" section, and the "Fixed" section can be removed. You should still leave a reference to the issue. Also, this should all be listed under an "Unreleased" section. Here's an updated version, to make your life a bit easier:

```
## Unreleased

**_WARNING!_ Default `GET /api/logout` endpoint is deprecated and will be removed in v1.0. You 
should use `POST /api/logout` instead. See #100**

#### Changed
- Default logout endpoint uses `POST` instead of `GET` (#100)


...
```